src/core/server/integration_tests/saved_objects/migrations/group3/migration_from_older_v1.test.ts

```diff
@@ -125,14 +126,24 @@ describe('migrating from 7.3.0-xpack which used v1 migrations', () => {
       .getTypeRegistry()
       .getAllTypes()
       .reduce((versionMap, type) => {
-        const { name, migrations, convertToMultiNamespaceTypeVersion } = type;
+        const { name, migrations, convertToMultiNamespaceTypeVersion, modelVersions } = type;
         if (migrations || convertToMultiNamespaceTypeVersion) {
           const migrationsMap = typeof migrations === 'function' ? migrations() : migrations;
           const migrationsKeys = migrationsMap ? Object.keys(migrationsMap) : [];
           if (convertToMultiNamespaceTypeVersion) {
             // Setting this option registers a conversion migration that is reflected in the object's `typeMigrationVersions` field
             migrationsKeys.push(convertToMultiNamespaceTypeVersion);
           }
+
+          const modelVersionCreateSchemas =
```

Comment:
Needed to adjust this test, since I've added [modelVersion](https://github.com/elastic/kibana/pull/183986/files#diff-8516da275ceca0b60b120667736b34b9ad6fb815e7001592f225a417ce501aa3R34-R50) for new `solution` property. Test didn't take into account schema changes coming from `modeVersion` that affect `typeMigrationVersions` field.